templates/shortcode-form-grid.php

```diff
@@ -378,41 +378,40 @@ static function($term)use($tag_text_color,$tag_bg_color){
-                                                '<span class="amount">%1$s</span>
-                                                        <span class="goal">of %2$s ' . $goalText . '</span>',
+                                                __('<span class="amount">%1$s</span>
+                                                    <span class="goal">of %2$s %3$s </span>', 'give'),
                                                 $form->get_sales(),
-                                                give_format_amount( $goal, array( 'decimal' => false ) ));
+                                                give_format_amount( $goal, array( 'decimal' => false ) ),
+                                                _n('donation', 'donations',  $form->get_sales(), 'give'));
 
                                         elseif ( 'donors' === $goal_format ) :
 
-                                            $goalText = _n('donor', 'donors',  $goal, 'give');
-
                                             echo sprintf( /* translators: 1: total number of donors completed 2: total number of donors set as goal */
-                                                '<span class="amount">%1$s</span>
-                                                        <span class="goal">of %2$s ' . $goalText . '</span>',
+                                                __('<span class="amount">%1$s</span>
+                                                    <span class="goal">of %2$s %3$s</span>','give'),
                                                 give_get_form_donor_count( $form->ID ),
-                                                give_format_amount( $goal, array( 'decimal' => false ) )
+                                                give_format_amount( $goal, array( 'decimal' => false ) ),
+                                                _n('donor', 'donors',  $goal, 'give')
                                             );
                                         endif;
                                         ?>
                                         </div>
 
                                         <div class="form-grid-raised__details">
                                             <?php
-                                            $amountText = _n('donation', 'donations',  $form->get_sales(), 'give');
 
-                                            echo sprintf(
-                                                '<span class="amount form-grid-raised__details_donations">%s</span>
-                                                        <span class="goal">%s</span>', $form->get_sales(), $amountText ) ?>
+                                            echo sprintf( /* translators: 1: total number of donations completed */
+                                                __('<span class="amount form-grid-raised__details_donations">%1$s</span>
```

Comment:
This HTML partial doesn't seem to need to be translatable. The `__n()` call below translates the only translatable text in the string.

Reply:
@kjohnson I ended up refactoring the sprintfs inside the tags to make it easier to read in https://github.com/impress-org/givewp/pull/6589/commits/1664d81f0cf0a12b9def3dc3fd65d4f1f6a60c35

<img width="1595" alt="Screen Shot 2022-10-20 at 9 56 32 AM" src="https://user-images.githubusercontent.com/75056371/197012121-a959daae-264c-4121-994b-150e751f44ca.png">